tests/wasm/no_shims.rs

```diff
@@ -133,14 +133,14 @@ fn no_shims() {
     outgoing_f32(y);
 
     let z = incoming_f64();
-    assert_eq!(z, 13.37);
+    assert_eq!(z, std::f64::consts::PI);
     outgoing_f64(z);
 
     let w = many(x, y, z);
     assert_eq!(w, 42);
 
     let pi = incoming_namespaced();
-    assert_eq!(pi, 3.14);
+    assert_eq!(pi, 13.37);
```

Comment:
Now I am just confused :laughing:.

Reply:
`3.14` was used as an arbitrary value, and the lint suggested to use the correct PI constant. I thought that was a great idea (as a precision test), but not specifically for the namespaced test. Therefore, I swapped the PI constant to the regular incoming/outgoing test and used 13.37 as a simpler value for the namespaced test.

Reply:
I see.
Using `Math.PI` was definitely a nice change.
But `outgoing_namespaced` specifically used the parameter name `pi`, which you changed to 13.37, so that weird.

I think it's perfectly fine like that, just change the parameter name of `outgoing_namespaced` to something else then `pi`.

Reply:
Nice!